CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0
```

Comment:
The version should actually be `0.1.3` (since there is a leading zero the rest of the numbers are shifted to the right)

Reply:
Changed.